src/plugin.json

```
@@ -17,19 +17,19 @@
     "links": [
       {
         "name": "Github",
-        "url": "https://github.com/Vonage/Grafana_Status_panel"
+        "url": "https://github.com/Grafana/Grafana_Status_panel"
       },
       {
         "name": "License",
-        "url": "https://github.com/Vonage/Grafana_Status_panel/blob/master/LICENSE"
+        "url": "https://github.com/Grafana/Grafana_Status_panel/blob/master/LICENSE"
       }
     ],
     "screenshots": [],
     "version": "2.0.0",
-    "updated": "2022-08-19"
+    "updated": "2024-05-03"
```

```suggestion
    "updated": "%TODAY%"
```
Any reason not to use ^ and %VERSION% above?

---

nope, no reason not to. updated